### DISCUSSION TITLE
Please Help on How to contact MF Support!!

### DISCUSSION THREAD

**Discussion by Sprolkyman** [MobiFlight/MobiFlight-Connector#1047] [Q&A]
I can no longer log in to Mobiflight website. 
I tried to create a new sign in, with the Email adress i always use, but get a message telling me to contact Mobiflight support for help because there is a duplicate email adress..!!

I cant find any support other then the forums, But i cant log in, so i can not post either! 

Does anyone know how to get in contact with MF Support?
 Thanks! David.

**Comment by elral** [comment]
If you scroll down to the bottom of the website you will find the imprint, and there is the email listed.
Otherwise register on Discord

**Comment by Sprolkyman** [comment]
Thank you for your reply! 
David.
